### PR TITLE
Terraform install latest 

### PR DESCRIPTION
**Pull request by acidsennui**
pt: https://www.pivotaltracker.com/story/show/159699819

### CODE REVIEW

File: terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash [hunk 1]
```diff
@@ -26,15 +26,12 @@ sudo service datadog-agent start
...
 #INSTALL EPOCH
 
-VERSION="${epoch_version}"
+URL="${epoch_package}"
 
-FILE="epoch-$VERSION-ubuntu-x86_64.tar.gz"
-URL="https://github.com/aeternity/epoch/releases/download/v$VERSION/$FILE"
-
-wget -q $URL -O /home/epoch/$FILE
+wget -q $URL -O /home/epoch/epoch.tar.gz
```
Comment (lucafavatella) [inline terraform/modules/cloud/aws/deploy/fleet/templates/user_data.bash]: Above, value of variable `URL` is assigned with quotes `"` while here no quotes used: shall quotes be here too?

I see name is inject from other config file, is URL, and potentially may contain strange characters. Though it does not matter for now so looks ok.

File: terraform/main.tf [hunk 2]
```diff
@@ -38,7 +38,7 @@ module "aws_deploy-ap-southeast-1" {
   instance_type = "m4.large"
 
   epoch = {
-    version = "0.19.0"
+    package = "https://s3.eu-central-1.amazonaws.com/aeternity-epoch-releases/epoch-latest-ubuntu-x86_64.tar.gz"
```
Comment (lucafavatella) [inline terraform/main.tf]: How does guarantee comes that `s3.eu-central-1.amazonaws.com` serves latest version of file? (I mean - eu-central-1 may have stale copy while another asia or so have a recent one.)

Comment (lucafavatella) [inline terraform/main.tf]: Does this terraform integration print the hash of downloaded artefact (minimum)?

Comment (michalbagrowski) [reply terraform/main.tf]: we out our images only in eu-central-1, aws it self to not synch it cross region

Comment (michalbagrowski) [reply terraform/main.tf]: terrafrom do not download this file, it's isntance that do this on boot, can add this there

Comment (acidsennui) [reply terraform/main.tf]: we upload image just to eu-central-1 s3 bucket, aws it self do not synch it to other regions

Comment (acidsennui) [reply terraform/main.tf]: terraform it self do not touch this file, it's used with `user_script` in boot of instance, i can add there hash if required

Comment (lucafavatella) [reply terraform/main.tf]: Not required. (I see it is release bucket so not many re-upload.)

**Comment by dincho** [comment]
> This seems to depend on aeternity/epoch#1437 so cannot be merged now?

Actually it does not until next release, because the `latest` artefact is manually put in place already.

**Pull request merged**